SECURITY.md

```diff
@@ -0,0 +1,44 @@
+# Security
+
+Thanks for helping make Parsec safe for everyone.
+
+We take the security of our software products and services seriously, including
+all of the open source code repositories managed through our GitHub
+organization, [Scille](https://github.com/Scille/).
+
+If you believe you have found a security vulnerability in any Scille-owned
+repository, please report it to us through coordinated disclosure as described
+below.
+
+## Disclosure policy
+
+**Please do not report security vulnerabilities through public GitHub issues,**
+**discussions, or pull requests.**
+
+Instead, please send an email to <security@parsec.cloud>.
```

Comment:
```suggestion
Instead, please send an email to [security@parsec.cloud](mailto:security@parsec.cloud).
```

Should the link explicitly use `mailto:...` ? 
Github seems to handle email address as link just fine :thinking: